src/tablib/formats/_xlsx.py

```diff
@@ -3,12 +3,12 @@
 
 from io import BytesIO
 
-import openpyxl
 import tablib
-
-Workbook = openpyxl.workbook.Workbook
-ExcelWriter = openpyxl.writer.excel.ExcelWriter
-get_column_letter = openpyxl.utils.get_column_letter
+from openpyxl.reader.excel import ExcelReader, load_workbook
+from openpyxl.styles import Alignment, Font
+from openpyxl.utils import get_column_letter
+from openpyxl.workbook import Workbook
+from openpyxl.writer.excel import ExcelWriter
```

Comment:
I don't mind these changes, although it does take away some context when they're used below.

Reply:
Good point!